Sming/Arch/Esp8266/Components/esp8266/ESP8266_NONOS_SDK.patch

```diff
@@ -0,0 +1,6 @@
+diff --git a/lib/libc.a b/lib/libc_.a
+rename from lib/libc.a
+rename to lib/libc_.a
+diff --git a/lib/libgcc.a b/lib/libgcc_.a
```

Comment:
Why is the underscore renaming here?

Reply:
Renaming them eliminates the risk of their being mistakely picked up instead of the newlib ones.

Reply:
Ok, can you rename them to lib(x).a -> lib(x).orig.a  instead of underscore?